common-java-libs/org.wso2.apk.apimgt.impl/src/main/java/org/wso2/apk/apimgt/impl/APIConsumerImpl.java

```diff
@@ -2720,7 +2720,14 @@ public ApiTypeWrapper getAPIorAPIProductByUUID(String uuid, String organization)
                     apiProduct = addTiersToAPI(apiProduct, organization);
                     return new ApiTypeWrapper(apiProduct);
                 } else {
-                    API api = APIMapper.INSTANCE.toApi(devPortalApi);
+                    //API api = APIMapper.INSTANCE.toApi(devPortalApi);
```

Comment:
Did you keep this comment intentionally?

Reply:
Yes, added a TODO task